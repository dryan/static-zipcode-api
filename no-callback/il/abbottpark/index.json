{"region": {"fips": "17", "name": "Illinois", "abbr": "IL"}, "postal_codes": ["60064"], "name": "Abbott Park", "counties": [{"fips": "097", "name": "Lake"}]}