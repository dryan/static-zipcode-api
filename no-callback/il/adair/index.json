{"region": {"fips": "17", "name": "Illinois", "abbr": "IL"}, "postal_codes": ["61411"], "name": "Adair", "counties": [{"fips": "109", "name": "McDonough"}, {"fips": "057", "name": "Fulton"}]}