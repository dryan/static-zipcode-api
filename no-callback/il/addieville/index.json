{"region": {"fips": "17", "name": "Illinois", "abbr": "IL"}, "postal_codes": ["62214"], "name": "Addieville", "counties": [{"fips": "189", "name": "Washington"}]}