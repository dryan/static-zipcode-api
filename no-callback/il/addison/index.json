{"region": {"fips": "17", "name": "Illinois", "abbr": "IL"}, "postal_codes": ["60101"], "name": "Addison", "counties": [{"fips": "043", "name": "Du Page"}]}