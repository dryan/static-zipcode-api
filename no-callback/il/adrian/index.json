{"region": {"fips": "17", "name": "Illinois", "abbr": "IL"}, "postal_codes": ["62330"], "name": "Adrian", "counties": [{"fips": "067", "name": "Hancock"}, {"fips": "071", "name": "Henderson"}]}