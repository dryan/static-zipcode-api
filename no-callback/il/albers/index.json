{"region": {"fips": "17", "name": "Illinois", "abbr": "IL"}, "postal_codes": ["62215"], "name": "Albers", "counties": [{"fips": "027", "name": "Clinton"}]}