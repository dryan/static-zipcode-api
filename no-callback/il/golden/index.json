{"region": {"fips": "17", "name": "Illinois", "abbr": "IL"}, "postal_codes": ["62339"], "name": "Golden", "counties": [{"fips": "001", "name": "Adams"}]}